packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -78,6 +78,7 @@ export const makeZoeSeatAdminKit = (
       updater.fail(reason);
       doExit(zoeSeatAdmin);
     },
+    getNotifier: () => Promise.resolve(notifier),
```

Comment:
would this be a good time to make it `getSubscriber` instead? we're moving away from Notifier in core code since consumers can `makeNotifierFromSubscriber` at will.

Reply:
I'd rather that were a separate task.